src/commands/info.rs

```diff
@@ -19,6 +19,11 @@ pub async fn info(ctx: &client::Context, msg: &Message, mut args: Args) -> Comma
 
     let color = member.colour(&ctx).await;
 
+    let mut color_roles = String::new();
+    for i in &member.roles {
+        color_roles.push_str(&format!("{} (#{}) ", i.mention(), i.to_role_cached(&ctx).await.context("couldn't get the cached role")?.colour.hex().to_string()));
```

Comment:
imo the function should really not fully fail if it fails to get the role for some reason, some more "lenient" way of handlign failure here would be good. also, plox split this up into multiple lines or at least run the formatter ^^